src/content/basics/azure-architecture/index.md

```diff
@@ -1,10 +1,11 @@
 +++
 title = "The Giant Swarm Azure Architecture"
 description = "Architecture Overview showing how Giant Swarm is set up on Microsoft Azure"
-date = "2019-07-16"
+date = "2019-11-21"
 weight = 20
 type = "page"
 categories = ["basics"]
+last review date = "2019-11-25"
```

Comment:
```suggestion
last_review_date = "2019-11-25"
```

I think this is a great addition but it actually breaks the automated tests! Can you try it with this change?

Reply:
My plan is to add that - slowly - to all the documents, in order to add sanity to the review cycle. If it breaks something then we'll have to think of something else.

Reply:
@Oshratn The problem is the spaces in `last review date` please try it again with underscores like in my suggestion. That should fix it.

Reply:
Gotcha, will do.